src/components/ForecastDetails.js

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import PropTypes from "prop-types";
+import WeatherIcon from "react-icons-weather";
+import "../styles/ForecastDetails.css";
+import highTempIcon from "../styles/images/high.png";
+import lowTempIcon from "../styles/images/low.png";
+import windIcon from "../styles/images/wind.png";
+import humidityIcon from "../styles/images/humidity.png";
+
+function ForecastDetails({ forecast }) {
+  const { date, temperature, humidity, wind, icon, description } = forecast;
+  const formattedDate = new Date(date).toLocaleDateString("en-uk", {
+    weekday: "short",
+    day: "numeric",
+    month: "short",
+  });
+  return (
+    <div className="forecast-details">
+      <div className="forecast-details-container-1">
+        <div className="forecast-details__date">{formattedDate}</div>
+        <div className="forecast-details__icon">
+          <WeatherIcon name="owm" iconId={icon} />
+        </div>
+        <div className="forecast-details__description">{description}</div>
+      </div>
+      <div className="forecast-details-container-2">
+        <div className="forecast-details__max-temperature">
+          <img
+            src={highTempIcon}
+            alt="high-temp logo"
+            width="25px"
+            height="25px"
```

Comment:
Theres no need to hard code values for css like this, these should all be removed from the JSX and instead be added into the CSS file. You could do something like:

```
.forecast-details__humidity img,
.forecast-details__max-temperature img,
.forecast-details__low-temperature img {
  height: 25px;
  width: 25px;
}

.forecast-details__wind img {
  height: 30px;
  width: 30px;
}
```